shared/wallets/wallet/settings/index.tsx

```diff
@@ -168,6 +197,46 @@ class AccountSettings extends React.Component<SettingsProps> {
               )}
             </Kb.Box2>
             <Divider />
+            {!!props.showExternalPartners && (
+              <Kb.Box>
+                <Kb.Box2
+                  direction="vertical"
+                  style={styles.section}
+                  fullWidth={true}
+                  gap="tiny"
+                  gapEnd={true}
+                >
+                  <Kb.Text type="BodySmallSemibold">External tools and partners</Kb.Text>
+                  {props.externalPartners.map((partner, index, array) => {
+                    const showDivider = index !== array.length - 1
```

Comment:
if you do the opposite and put the divider in front you don't need to use the array at all, just use index > 0